recipes/metis/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "metis" %}
+{% set version = "4.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: metis-4.0.3.tar.gz
+  md5: d3848b454532ef18dc83e4fb160d1e10
+  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD/metis-4.0.3.tar.gz
+
+build:
+  skip: true  # [win]
+  number: 0
+  binary_relocation: true
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+
+test:
+  commands:
+    - graphchk
+    - partnmesh
+    - kmetis
+    - mesh2dual
+    - mesh2nodal
+    - oemetis
+    - onmetis
+    - partdmesh
+    - pmetis
+
+about:
+  home: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview
+  license: GPLv3
```

Comment:
So the license for 4.0.3 appears to be pretty terrible and not at all GPL. Basically it can only be used by non-profits and US government agencies only for research and educational usage. Others can evaluate it, but not depend on it.

There is also some question as to whether we can distribute it. See this line below copied from the license. We certainly are not selling it, but we may be distributing it depending on how they chose to define that. Not sure how SuiteSparse handled this issue (perhaps just by making it optional).

>  One may make copies of the software for their use provided that the copies, are not sold or distributed, are used under the same terms and conditions.


Have quoted the rest of the license below so that it can be seen.

```
The METIS package is copyrighted by the Regents of the University of
Minnesota. It can be freely used for educational and research purposes 
by non-profit institutions and US government agencies only. Other 
organizations are allowed to use METIS only for evaluation purposes,
and any further uses will require prior approval. The software 
may not be sold or redistributed without prior approval. One may 
make copies of the software for their use provided that the copies, 
are not sold or distributed, are used under the same terms and 
conditions. 

As unestablished research software, this code is provided on an 
``as is'' basis without warranty of any kind, either expressed or 
implied. The downloading, or executing any part of this software 
constitutes an implicit agreement to these terms. These terms and 
conditions are subject to change at any time without prior notice.
```